tamagui.config.ts

```diff
@@ -1,7 +1,7 @@
 import { config as configBase } from '@tamagui/config/v3'
 import { createTamagui } from 'tamagui'
 
-import { fontConfig } from '~/fonts'
+import { fontConfig } from './fonts'
```

Comment:
Why not a relative path? 
import { fontConfig } from '~/fonts'

Reply:
When using a relative path here, an error may occur: "Could not resolve path."